src/components/ScriptExplorer/SignatureImporter.jsx

```diff
@@ -576,8 +596,17 @@ function mapStateToProps(state, ownProps) {
       signatureImporter: state.spend.signatureImporters[ownProps.number],
       fee: state.spend.transaction.fee,
       txid: state.spend.transaction.txid,
+      walletName: state.wallet.common.walletName,
     },
     ...state.spend.transaction,
+    requiredSigners: state.settings.requiredSigners,
+    extendedPublicKeys: Object.values(
+      state.quorum.extendedPublicKeyImporters
+    ).map((key) => ({
+      bip32Path: key.bip32Path === "Unknown" ? "m/45'/0/0/0" : key.bip32Path,
```

Comment:
Having seen this earlier... was in lowercase? Might want to double check or clamp to check.